rclpy/test/test_rate.py

```diff
@@ -53,7 +56,9 @@ def _run(self, rate):
                 last_wake_time = now
 
             self.avg_period = sum(measurements) / len(measurements)
-            self.max_jitter = max([abs(m) - self.avg_period for m in measurements])  # noqa: C407
+            self.max_jitter = max([abs(m - self.avg_period) for m in measurements])  # noqa: C407
```

Comment:
The change to the formula looks good to me. The test still seems to fail on macOS and Windows though.

Reply:
It's a flakey test on those systems, see (https://github.com/ros2/rclpy/issues/513). I'm really curious why windows sees jitter with such very precise rate periods, but I can open a PR that doubles the max_jitter for windows/macos if we want to improve the reliability of this test.

Reply:
I can't really explain why, but its common that we have to relax the timing requirement on tests for them to pass on Windows.  Another recent example is in https://github.com/ros2/rclcpp/pull/993/files#diff-0ca2dd153932c387ce77cd330d3c70c6 .